spec-tests/src/test/java/org/candlepin/spec/consumers/ConsumerResourceActivationKeySpecTest.java

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright (c) 2009 - 2022 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.spec.consumers;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.candlepin.dto.api.client.v1.ActivationKeyDTO;
+import org.candlepin.dto.api.client.v1.ConsumerDTO;
+import org.candlepin.dto.api.client.v1.ContentOverrideDTO;
+import org.candlepin.dto.api.client.v1.OwnerDTO;
+import org.candlepin.resource.client.v1.ActivationKeyApi;
+import org.candlepin.spec.bootstrap.client.ApiClient;
+import org.candlepin.spec.bootstrap.client.ApiClients;
+import org.candlepin.spec.bootstrap.client.api.ConsumerClient;
+import org.candlepin.spec.bootstrap.client.api.OwnerClient;
+import org.candlepin.spec.bootstrap.data.builder.ConsumerTypes;
+import org.candlepin.spec.bootstrap.data.builder.Consumers;
+import org.candlepin.spec.bootstrap.data.builder.Owners;
+import org.candlepin.spec.bootstrap.data.util.StringUtil;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Set;
+
+
+public class ConsumerResourceActivationKeySpecTest {
+
+    static ApiClient client;
+    static OwnerClient ownerClient;
+    static ConsumerClient consumerClient;
+    static OwnerDTO owner;
+    static ActivationKeyApi activationKeyApi;
+
+
+    @BeforeAll
+    static void beforeAll() {
+        client = ApiClients.admin();
+        ownerClient = client.owners();
+        consumerClient = client.consumers();
+        owner = Owners.random();
```

Comment:
Let's not make the owner globally available as per the guidelines. There's only 1 test for now, but the rest of them will be ported at some point